_rules/target-size-ua-control-vcup8d.md

```diff
@@ -0,0 +1,136 @@
+---
+id: vcup8d
+name: Interactive component has size controlled by User Agent
+rule_type: atomic
+description: |
+  This rule checks that elements that can receive pointer events have a size controlled by the user agent.
+accessibility_requirements:
+  wcag21:2.5.5: # Target size (enhanced) (AAA)
+    secondary: 'This success criterion is **less strict** than this rule. This is because the rule does not consider the size of the elements. Some of the failed examples may satisfy this success criterion.'
+  wcag22:2.5.8: # Target Size (Minimum) (AA)
+    secondary: 'This success criterion is **less strict** than this rule. This is because this criterion has a lower size requirement. Some of the failed examples may satisfy this success criterion.'
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+---
+
+## Applicability
+
+This rule applies to any [HTML element][namespaced element] which [can be targeted by a pointer event][].
+
+## Expectation
+
+Each test target is a [User Agent controlled component][].
+
+## Assumptions
+
+- This rule assumes that [focusable][] `widget` are effectively clickable. If a widget is [focusable][] without being clickable, it may fail this rule while [Success Criterion 2.5.5 Target Size (enhanced)][sc255] and [Success Criterion 2.5.8 Target Size (minimum)][sc258] are satisfied.
+
```

Comment:
We already have secondary requirements, but should we clarify, once more, that "User Agent controlled components" are not required by WCAG, and the author can still customize them? 

I’m concerned because the failing examples (especially the second one) currently meet 2.5.8, and I don’t want a distracted reader to think that custom controls are not allowed.